geode-core/src/main/java/org/apache/geode/distributed/internal/LonerDistributionManager.java

```diff
@@ -1054,14 +1055,14 @@ public boolean awaitTermination(long timeout, TimeUnit unit) throws InterruptedE
 
     @Override
     public <T> Future<T> submit(final Callable<T> task) {
-      Exception ex = null;
-      T result = null;
+      CompletableFuture future = new CompletableFuture<>();
```

Comment:
This new code looks like an improvement. But the method is on `DummyExecutor`: question: is this class used? Because I don't see any static usages of the class.

Can we remove `DummyExecutor` entirely?

Reply:
yes!  done!